### PR TITLE
Fix Array comparision types to pass with Node.js >= 12

### LINKED ISSUE #179: Test fails with Node.js ≥ 12
Hi,
when trying to launch `npm run test` with Node.js ≥ 12, test fails because `deepEqual` compares Utf8Array to Array.

### PR DESCRIPTION
**Pull request by nileshpatra**
Fixes #179

### CODE REVIEW

**Comment by nileshpatra** [comment]
@puzrin Just changed the type comparision.
Could you take a review if it looks OK?
That would be great!

File: test/helpers.js [hunk 1]
```diff
@@ -66,6 +66,8 @@ function testSingle(zlib_method, pako_method, data, options) {
   // position (= no additional gzip headers used)
   if (options.ignore_os) zlib_result[9] = pako_result[9];
 
+  zlib_result = Array.from(Uint8Array);
+  pako_result = Array.from(Uint8Array);
```
Comment (puzrin) [inline test/helpers.js]: 1. I don't understand why this should work at all. Looks like data will be missed.
2. I would cast to typed array. Ordinary arrays are legacy for ancient browsers, id doesn't worth spread those without need.

Comment (nileshpatra) [reply test/helpers.js]: Yes, it's kind of urgent. We need pako to pass with Node.js 12. It would be great to have it fixed ASAP.

**Comment by nileshpatra** [comment]
@puzrin it's urgent for me to be fixed. Could you please fix this?
That would be great (humble request)

Thanks!

**Comment by puzrin** [comment]
https://github.com/nodeca/pako/commit/2fa56efc8f75642ffc2f03440a0269830978c8b6

**Comment by nileshpatra** [comment]
@puzrin Many thanks!

**Pull request closed without merging**